sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -195,4 +196,19 @@ class SQLQuerySuite extends QueryTest {
     checkAnswer(sql("SELECT sum( distinct key) FROM src group by key order by key"),
       sql("SELECT distinct key FROM src order by key").collect().toSeq)
   }
+
+  test("SPARK-4963 SchemaRDD sample on mutable row return wrong result") {
+    sql("CREATE TABLE IF NOT EXISTS src (key INT, value STRING)")
+    val location =
+      Utils.getSparkClassLoader.getResource("data/files/kv1.txt").getFile()
+    sql(s"LOAD DATA LOCAL INPATH '$location' INTO TABLE src")
```

Comment:
You don't need to create the `src` table.  Our test harness does that automatically whenever the test tables are referenced.  I can remove this when merging.